README.md

```diff
@@ -136,21 +136,24 @@ Default: `null`
 
 Type: `str`
 
-### `mssql_input_sql_file`
+### `mssql_pre_input_sql_file` and `mssql_post_input_sql_file`
```

Comment:
State that `mssql_input_sql_file` is deprecated and that `mssql_post_input_sql_file` should be used instead.

Reply:
Fixed in README. I've also added a debug module above to print a deprecated warning.